services/static-webserver/client/source/class/osparc/dashboard/ServiceBrowser.js

```diff
@@ -112,23 +112,57 @@ qx.Class.define("osparc.dashboard.ServiceBrowser", {
     },
 
     __itemClicked: function(card) {
-      // const key = card.getUuid();
-      // this._createStudyFromService(key, null);
       const serviceData = card.getResourceData();
       this._openDetailsView(serviceData);
       this.resetSelection();
     },
 
-    _createStudyFromService: function(key, version) {
+    _createStudyFromService: async function(key, version) {
       if (!this._checkLoggedIn()) {
         return;
       }
 
+      const isDevel = osparc.utils.Utils.isDevelopmentPlatform();
```

Comment:
@matusdrobuliak66 all API entries connected to this functionality should be enabled/disabled using `WEBSERVER_DEV_FEATURES_ENABLED`

Reply:
@matusdrobuliak66 Notified 👌